anycloud/docs/how-to/aws-postgres.md

```diff
@@ -0,0 +1,132 @@
+
+## Create a PostgreSQL instance on AWS RDS
+
+Follow this [AWS guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html#CHAP_GettingStarted.Creating.PostgreSQL) to create a PostgreSQL on AWS RDS.
+
+## Enable programmatic AWS access to VMs for AnyCloud
+
+1) Create a new IAM user in your AWS account using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html#id_users_create_console).
+
+2) Create a new access key under that IAM user using their console/UI as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey).
+
+3) Enable programmatic access for that IAM user, and attach the built-in [`AmazonEC2FullAccess`](https://console.aws.amazon.com/iam/home#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FAmazonEC2FullAccess) policy to it as described [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html#add-policies-console).
+
+4) Take the `accessKeyId` and `secretAccessKey` from step 2 and create AWS `Credentials` in AnyCloud. You will need to pick a name or alias for the `Credentials`. The initial value will be `aws`. In this example, we will call it `mystartup-aws`.
+
+```bash
+$ anycloud credentials new
+? Pick cloud provider for the new credentials ›
+❯ AWS
+  GCP
+  Azure
+Name for new Credentials: mystartup-aws
+AWS Access Key ID: ******************
+AWS Secret Access Key: ******************
+Successfully created "mystartup-aws" Credentials
+```
+
+## Deploy the REST API using AnyCloud and link it to the PostgreSQL instance
+
+1) First download Prisma's example and install dependencies
+
+```
+curl https://codeload.github.com/prisma/prisma-examples/tar.gz/latest | tar -xz --strip=2 prisma-examples-latest/typescript/rest-express
+mv rest-express anycloud-node-aws-psql
+```
+
+Install npm dependencies:
+
+```
+cd anycloud-node-aws-psql
+npm install
+```
+
+2) Add `npm scripts` to the `package.json` file
+
+```
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf dist",
+    "start": "node ./dist/src/index.js",
+  }
+```
+
+3) Define the `Dockerfile`: 
+
+```bash
+FROM node:lts
+
+COPY . .
+
+RUN npm install
+RUN npm run build
+CMD npm run start
+```
+
+4) Initialize a `git` repository
+
+```bash
+git init
+git add -A
+git commit -m "Initial commit"
+```
+
+5) Use the AnyCloud CLI to create an `anycloud.json` file in the project directory and define a `Deploy Config`. You will need to pick a name or alias for the `Deploy Config`. The initial value will be `staging`. Since there is only `Credentials`, the CLI will default to that for your new `Deploy Config`.
```

Comment:
Also, should not be `Since there is only one 'Credential' defined, ...`? or `If you only have one 'Credential' defined, ...`

Reply:
> Also, should not be `Since there is only one 'Credential' defined, ...`? or `If you only have one 'Credential' defined, ...`

Convention that @dfellis pointed out is that credentials is used over its singular form credential. AWS and most other cloud providers also use the plural form.